Sources/Attribution/AttributionNetwork.swift

```diff
@@ -22,37 +22,43 @@ import Foundation
     /**
      Apple's search ads
      */
-    case appleSearchAds
+    @available(*, deprecated, message: "use adServices")
```

Comment:
Maybe this could use some more explanation.

Reply:
Or replace with `renamed: "adServices"`

Reply:
idk i hesitate since it's not really a rename, it results in different data being collected. but at the same time, the auto-correction is nice.

Reply:
Oh yeah that's a great point